src/api/schemas/AiAgentBasicTextTool.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
In the future, when we switch to TS-SDK, does this entire folder get replaced?

Reply:
Yes, probably. It makes sense already to have a separate space for models as there are quite many for this endpoint